Code/GraphMol/ChemTransforms/ChemTransforms.cpp

```diff
@@ -473,6 +473,13 @@ ROMol *replaceCore(const ROMol &mol, const ROMol &core,
       // add the bonds now, after we've finished the loop over neighbors:
       for (auto &newBond : newBonds) {
         newMol->addBond(newBond, true);
+        auto beginAtom = newBond->getBeginAtom();
```

Comment:
I like this addition, but I think it's worth putting this whole block inside a check to see if the molecule actually has any conformers.